runtime/mainnet/src/config/governance.rs

```diff
@@ -0,0 +1,548 @@
+	type ResetOrigin = AtLeastThreeFourthsOfCouncil;
+	type RuntimeEvent = RuntimeEvent;
+	type SwapOrigin = AtLeastThreeFourthsOfCouncil;
+	type WeightInfo = pallet_membership::weights::SubstrateWeight<Runtime>;
+}
+
+impl pallet_motion::Config for Runtime {
+	type RuntimeCall = RuntimeCall;
+	type RuntimeEvent = RuntimeEvent;
+	// SimpleMajorityOrigin won't ever ensure origin.
+	type SimpleMajorityOrigin = NeverEnsureOrigin<()>;
+	// At least 3/4 of the council vote is needed.
+	type SuperMajorityOrigin = AtLeastThreeFourthsOfCouncil;
+	// A unanimous council vote is needed.
+	type UnanimousOrigin = UnanimousCouncilVote;
+	type WeightInfo = pallet_motion::weights::SubstrateWeight<Runtime>;
+}
+
+#[cfg(test)]
+mod tests {
+	use std::any::TypeId;
+
+	use sp_runtime::traits::Get;
+
+	use super::*;
+
+	mod council_collective {
+		use frame_system::WeightInfo;
+		use pallet_collective::DefaultVote;
+
+		use super::*;
+
+		#[test]
+		fn consideration_is_not_configured() {
+			assert_eq!(
+				TypeId::of::<
+					<Runtime as pallet_collective::Config<CouncilCollective>>::Consideration,
+				>(),
+				TypeId::of::<()>(),
+			);
+		}
+
+		#[test]
+		fn default_vote_is_more_than_majority_then_prime() {
+			assert_eq!(
+				TypeId::of::<<Runtime as pallet_collective::Config<CouncilCollective>>::DefaultVote>(
+				),
+				TypeId::of::<pallet_collective::MoreThanMajorityThenPrimeDefaultVote>(),
+			);
+		}
+
+		#[test]
+		fn default_votes_unanimous_threshold_prime_is_nay() {
+			// Prime does not exist. (prime vote defaults to false)
+			let prime = None;
+			let seats: u32 = 5;
+			// We want unanimous consensus.
+			let thold = seats;
+
+			let aye_votes: u32 = 3;
+			let nay_votes: u32 = 0;
+			let abstentions = seats - (aye_votes + nay_votes);
+
+			let default = <Runtime as pallet_collective::Config<CouncilCollective>>::DefaultVote::default_vote(prime, aye_votes, nay_votes, seats);
+			assert_eq!(default, true);
+			// Abstentions will be added to aye votes.
+			// Threshold will be met and the proposal approved.
+			assert!(aye_votes + abstentions >= thold);
+
+			let aye_votes: u32 = 3;
+			let nay_votes: u32 = 1;
+			let abstentions = seats - (aye_votes + nay_votes);
+
+			let default = <Runtime as pallet_collective::Config<CouncilCollective>>::DefaultVote::default_vote(prime, aye_votes, nay_votes, seats);
+			assert_eq!(default, true);
+			// Abstentions will be added to aye votes.
+			// Threshold won't be met and the proposal disapproved.
+			assert!(!aye_votes + abstentions >= thold);
+
+			let aye_votes: u32 = 2;
+			let nay_votes: u32 = 1;
+			let abstentions = seats - (aye_votes + nay_votes);
+
+			let default = <Runtime as pallet_collective::Config<CouncilCollective>>::DefaultVote::default_vote(prime, aye_votes, nay_votes, seats);
+			assert_eq!(default, false);
+			// Abstentions will be added to nay votes.
+			// Threshold won't be met and the proposal disapproved.
+			assert!(!aye_votes + abstentions >= thold);
```

Comment:
If they are added to nay votes it shouldn't be summed right?
```suggestion
			// Abstentions will be added to nay votes.
			// Threshold won't be met and the proposal disapproved.
			assert!(!aye_votes >= thold);
```

We are doing some side math without testing anything with these tests. It provides some nice extra info about when a proposal passes or not but I don't think they should be added to these tests. Tests should be as simple and specific as possible. Threshold logic should be tested in the pallet.

Note: I hope that I didn't miss anything, otherwise ignore me.

Reply:
You don't need to test it because it should already be tested in the collectives pallet :)

Reply:
Yeah, the goal was not really testing, but having the info handy. 

Noise is gone now: [d86f49f](https://github.com/r0gue-io/pop-node/pull/448/commits/d86f49fead6fd39f3c4069979211601d8e141d58)